ecosystem/web-wallet/package.json

```diff
@@ -23,11 +23,15 @@
     "react-icons": "^4.3.1",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
-    "typescript": "^4.6.3"
+    "typescript": "^4.6.3",
+    "webextension-polyfill": "^0.9.0"
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "prebuild": "rimraf build",
+    "build": "npm-run-all build:*",
+    "build:app": "INLINE_RUNTIME_CHUNK=false react-scripts build",
+    "build:bg": "webpack --mode production ./src/background.ts --output-path ./build --output-filename background.ts",
```

Comment:
Some crazy thing so that background.ts has access to some src files before being made public. Metamask uses some more heavy tooling to do this aka "lavamoat" and some other things